doc/users/next_whats_new/2019-12-09-legend-textcolor.rst

```diff
@@ -0,0 +1,9 @@
+Legend now has a textcolor kwarg
+-------------------------------------
+
+The ``Axes.legend`` method now has a ``textcolor`` keywork argument. The 
```

Comment:
The text color of legend labels can now be set by passing a parameter ``textcolor`` to `.axes.Axes.legend`. It can ether be ...

Reply:
I used `:meth:`~.axes.Axes.legend``. Is that the best way to do this?

Reply:
You can leave out `:meth:` before because we use `'obj'` as the default sphinx role (https://matplotlib.org/devdocs/devel/documenting_mpl.html#referring-to-other-code). Other than that, this is the best way.

Reply:
Cool, I removed `:meth:`.